docs/section22/Rule22-42.md

```diff
@@ -0,0 +1,98 @@
+**Rule Assertion:** Baseline RMD = expected value  
+**Appendix G Section:** G3.2.2.1 Baseline 
+
+**Mandatory Rule:** True  
+**Evaluation Context:** Each Chiller  
+**Table Lookup:**
+J-6
+**Function Call:**  
+
+## Applicability Check:  
+- look at each chiller - only water-cooled chillers are applicable (ie chillers with a condensing_loop): `for chiller in B_RMD.chillers:`
+    - if the chiller has a condensing_loop, and the chiller energy source is electricity, continue to rule logic: `if chiller.consensing_loop != NULL and chiller.energy_source_type == "ELECTRICITY": CONTINUE TO RULE LOGIC`
+
+        ## Rule Logic:
+        - figure out the set of performance curves needed for the chiller - this should be V, X, Y, Z, AA or AB.  Initialize a variable curve_set and set it to "NONE": `curve_set = "NONE"`
+        - get the chiller rated capacity: `rated_capacity = chiller.rated_capacity`
+        - convert the chiller rated capacity from watts to tons: `chiller_capacity_tons = rated_capacity * 0.000284345`
+        - check if the compressor type is CENTRIFUGAL: `if chiller.compressor_type == "CENTRIFUGAL":`
+            - if the capacity is less than 150 tons, the category is V: `if chiller_capacity_tons < 150: curve_set = "V"`
+            - else if the capacity is greater than or equal to 300 tons, the category is Y: `elif chiller_capacity_tons >= 300: curve_set = "Y"`
+            - otherwise, the capacity is between 150 and 300, so the curves set is X: `else: curve_set = "X"`
+        - otherwise, check if the compressor type is POSITIVE_DISPLACEMENT or SCROLL: `if chiller.compressor_type in ["POSITIVE_DISPLACEMENT","SCROLL"]:`
+            - if the capacity is less than 150 tons, the category is Z: `if chiller_capacity_tons < 150: curve_set = "Z"`
+            - else if the capacity is greater than or equal to 300 tons, the category is AA: `elif chiller_capacity_tons >= 300: curve_set = "AA"`
+            - otherwise, the capacity is between 150 and 300, so the curves set is AB: `else: curve_set = "AB"`
+        - continue if the curve set is not "NONE" - the curve set could be none if the chiller.compressor_type is not one of the recognized types for Appendix G: `if curve_set <> "NONE":`
+            - calculate the chiller rated power by dividing the chiller rated capacity by the chiller full load efficiency (cop): `rated_power = chiller.rated_capacity / chiller.full_load_efficiency`
+            - create a list of the expected capacity validation points - this is in 25% intervals, starting with 0.25 to 1: `expected_validation_plr = [0.25,0.5,0.75,1]`
+            - create a list of the expected temperatures for the chilled water temperature for the validation calculations - expected_chwt_temps - units are (F).  The values are selected to capture the minimum, maximum, rating condition and some points in between: `expected_cwt_temps = [39,45,50,55]`
```

Comment:
Based on G3.2.3.9 the supply water temperature is reset up to a supply of 54F. Perhaps this is a good top of the range value then as opposed to using the 55F.

Reply:
Recommend expected_chwt_temps to make it easier to follow